packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -92,6 +133,10 @@ function isAzureMonorepoPackage(model: RLCModel): boolean {
   );
 }
 
+function isAzurePackage(model: RLCModel): boolean {
```

Comment:
I remember we have this logic already?

Reply:
let me check

Reply:
do you mean these two sperated ones: https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/buildPackageFile.ts#L88-L99

Reply:
Can you search all the other places that are using flavor === "azure" now? I wonder if we should make them consistent to leverage the helper here ?

Reply:
resolved.